zephyr_patches/0001-drivers-peci-xec-Fix-error-recovery-handling.patch

```diff
@@ -0,0 +1,58 @@
+From 7d3dde5e5e9ae7581741bfdf52e4edaf22b75063 Mon Sep 17 00:00:00 2001
+From: Rajavardhan Gundi <rajavardhan.gundi@intel.com>
+Date: Thu, 10 Sep 2020 11:26:59 +0530
+Subject: [PATCH] drivers: peci: xec: Fix error recovery handling
+
+Fix the error recovery mechanism that makes use of a temp variable
```

Comment:
I didn't know patches could be given like this. Very cool, and good that you added in readme.

Reply:
It's not ideal, but common practice in pre-integration